### ISSUE TITLE
FastX

### ISSUE THREAD

**Issue by DrSpringy** [UQ-RCC/ipp#96]
can we implement FastX or similar for better GUI performance when rendering 3D datasets?

**Comment by hoangnguyen177** [comment]
Definitely, it really depends on how open fastX is, and how open Irek is willing to make it :). 
Otherwise, we can consider other options, like RDP.

**Comment by markendr** [comment]
FastX demo from Edan before CVL?
OOD performance for 3D rendering?
-is it using VNC?
-any performance tuning options?
